getting-started/setup-multi-node-auth.md

```diff
@@ -445,3 +445,6 @@ docs:
 [detach_data_node]: /api#detach_data_node
 [distributed_exec]: /api#distributed_exec
 [configuration]: /getting-started/configuring
+[install]: /getting-started/installation
+[setup]:/getting-started/setup
```

Comment:
Is a space needed after `:`?  (I don't recall offhand, just flagging in case ;)

Reply:
Appears so: https://www.markdownguide.org/basic-syntax#formatting-the-second-part-of-the-link
> The label, in brackets, followed immediately by a colon and at least one space (e.g., [label]: ).